functions/index.js

```diff
@@ -0,0 +1,24 @@
+const functions = require('firebase-functions');
+const fetch = require('node-fetch');
+const HONESTY_STORE_LOGO = 'https://honesty.store/assets/android/icon@MDPI.png';
+const BOT_USERNAME = 'honesty.store';
+
+exports.sendSlackMessage = functions.https.onCall(data => {
+  const token = functions.config().react_app.slack_token;
+
+  const result = fetch(
+    `https://slack.com/api/chat.postMessage?token=${token}&channel=${
+      data.userid
```

Comment:
Not introduced in this work, but thinking this can get a bit hard to read with all the interpolation.

We could use the `query-string` package to encode an object and maybe make this a little clearer

```
const options = queryString.stringify({
  token,
  channel: data.userid,
  icon_url: HONESTY_STORE_LOGO,
  username: BOT_USERNAME,
  text: `Click to purchase your ${data.itemName}: https://honesty.store/item/${data.acutalItemID}`,
})

return fetch(`https://slack.com/api/chat.postMessage?${options}`)
  .then(...)
  .catch(...)
```

We could even extract the different urls to constants if we wanted to take this a step further

What do you think?